src/Core/Services/OpenAPI/OpenApiDocumentor.cs

```diff
@@ -166,90 +167,94 @@ private OpenApiPaths BuildPaths()
         {
             OpenApiPaths pathsCollection = new();
 
-            foreach (KeyValuePair<string, DatabaseObject> entityDbMetadataMap in _metadataProvider.EntityToDatabaseObject)
+            foreach (ISqlMetadataProvider metadataProvider in _metadataProviderFactory.ListMetadataProviders())
```

Comment:
We should NOT do this for a CosmosSqlMetadataProvider. CosmosDB_NoSql doesnt have REST enabled.

Reply:
discussed offline. same reasoning as below comment.